examples/jsm/controls/DragControls.js

```diff
@@ -37,7 +37,7 @@ class DragControls extends EventDispatcher {
 
 		this.rotateSpeed = 1;
 
-		//
+		this.mouseButtons = { LEFT: true, MIDDLE: true, RIGHT: true };
```

Comment:
The idea is to have an object that maps buttons to actions. E.g. `LEFT: MOUSE.ROTATE`. 

For `DragControls`, I would expect something like this:
```js
this.mouseButtons = { LEFT: MOUSE.DRAG, MIDDLE: MOUSE.DRAG, RIGHT: MOUSE.DRAG };
```

Reply:
The actions of the `MOUSE` enum are maintained here: https://github.com/mrdoob/three.js/blob/5a7ef89952256ce7fb9ae3401f27c5f12ec4e276/src/constants.js#L3